easy-ui-react/src/Button/Button.stories.mdx

```diff
@@ -72,13 +80,9 @@ export const ButtonStoryOnDarkBackgroundDecorator = (story) => (
   <Story name="OutlinedButtons" decorators={[ButtonStoryDecorator]}>
     <Button variant="outlined" />
     <Button color="secondary" variant="outlined" />
-    <Button color="success" variant="outlined" />
-    <Button color="warning" variant="outlined" />
```

Comment:
check with design, no use cases for these combinations and safe to consider as an invalid combination